security-scan/ui/styles.css

```
@@ -0,0 +1,1647 @@
+  border-collapse: collapse;
+  table-layout: fixed;
+}
+
+[data-iii-ui="security-scan"] .security-scan-ui-source-table caption {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)